jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -513,8 +513,8 @@ private static void checkToolSecurityPermssion() throws SecurityException {
 	}
 
 	private static void checkLegacySecurityPermssion() throws SecurityException {
-		if (!("false".equalsIgnoreCase(com.ibm.oti.vm.VM.getVMLangAccess()	//$NON-NLS-1$
-			.internalGetProperties().getProperty(LEGACY_DUMP_PERMISSION_PROPERTY)))) {
+		if (!("false".equalsIgnoreCase(com.ibm.oti.vm.VM //$NON-NLS-1$
+				.internalGetProperties().getProperty(LEGACY_DUMP_PERMISSION_PROPERTY)))) {
```

Comment:
The closing `) {` should be on e separate line.

Reply:
Updated